scripts/quarantine_stats/generate_dashboard.py

```
@@ -0,0 +1,1878 @@
+        RuntimeError: If git command fails.
+    """
+    try:
+        result = subprocess_run(
+            args=["git", "branch", "-r", "--list", "origin/main", "origin/cnv-*"],
+            capture_output=True,
+            text=True,
+            check=True,
+            cwd=cwd,
+        )
+    except CalledProcessError as error:
+        raise RuntimeError(f"Failed to get remote branches: {error.stderr}") from error
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)